pkg/kubecli/ovm.go

```diff
@@ -0,0 +1,119 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017 Red Hat, Inc.
+ *
+ */
+
+package kubecli
```

Comment:
Can you add tests for this package? Should be pretty much copy-paste from e.g. vm_test.go

Reply:
I am confused. You mean test the kubecli package or provide unit tests for ovm.go? Are ovm_tests.go next to ovm.go insuficient?

Reply:
The tests in `kubecli/*_test.go` do real rest calls (we start a go httptest server).

Reply:
Also the controller does not use every function from the client.